### DISCUSSION TITLE
Organizing session logs

### DISCUSSION THREAD

**Discussion by oshn** [archavka/aparat#5] [Ideas]
Currently we organize by date, e.g.:
- #3 
- #4 

What you think if we collect it by game, e.g.:
- **Well of Wilds Megaways** by Red Tiger
- **Time Machine** by Reflex Gaming

**Comment by archavka** [comment]
That is good idea
1. there are gaps between dates of our sessions 
2. we already have dates and time on screenshots

Let's heat up the wires for proper reason
